### ISSUE TITLE
Add monaco editor

### ISSUE THREAD

**Issue by jatinkatyal13** [coding-blocks/hacker-blocks#227]
Replace Ace Editor that we currently use with [Monaco Editor for Ember](https://github.com/mike-north/ember-monaco).

**Comment by shubhamofbce** [comment]
I would like to work on this issue!!

**Comment by sameshl** [comment]
@jatinkatyal13 I would like to like to work on this issue but would like some help from you. Can you please share some links with relevant info so I can get started with this issue.

**Comment by sudo-nan0-RaySK** [comment]
I am willing to take this!

**Comment by sudo-nan0-RaySK** [comment]
The version of ember-monaco you have suggested is no doubt maintained. However it has issues regarding CommonJS transforms. It is due to a node module named 'penpal'. The older [ember-monaco-editor](https://www.npmjs.com/package/ember-monaco-editor) has no such dependencies however. Should i go forward with it?

**Comment by abhishek97** [comment]
What issues exactly? We already are using that plugin online CodingBlocks frontend. Can you elaborate the issue or a link perhaps?

**Comment by sudo-nan0-RaySK** [comment]
Okay so when i install the above add-on, what i get is an error stating unknown transformation name 'cjs'. I have used the above add on in my project, it works fine in there but with hacker-blocks frontend I get [this](https://github.com/empress/empress-blog/issues/3) error. Only difference is the ember version.

**Comment by abhishek97** [comment]
Have you tried installing https://github.com/rwjblue/ember-cli-cjs-transform ?

**Comment by sudo-nan0-RaySK** [comment]
Yes, That's the first thing I did. Also the Monaco Editor for Ember add on uses it internally for importing Penpal.Like have a look at its index.js

**Comment by RAJAGOPALAN-GANGADHARAN** [comment]
@abhishek97  is this one open for working? I have experiences using ace editor so might be suitable for me
😄 
Thank you!
